From d209a09ad79761889363b14f379cce18716550b1 Mon Sep 17 00:00:00 2001
From: Mamoru TASAKA <mtasaka@fedoraproject.org>
Date: Wed, 28 May 2014 15:39:35 +0900
Subject: [PATCH 2/2] delaunay: don't use double-backslash comments

gcc -std=c89 makes double-backslash comments an error, so wrap this
with c89-supported comments
---
 hacks/delaunay.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/hacks/delaunay.c b/hacks/delaunay.c
index e57e77b..a9d3de5 100644
--- a/hacks/delaunay.c
+++ b/hacks/delaunay.c
@@ -87,9 +87,11 @@ circumcircle (double xp,double yp,
   dy = yp - *yc;
   drsqr = dx*dx + dy*dy;
 
+  /*
   // Original
   //return((drsqr <= *rsqr) ? TRUE : FALSE);
   // Proposed by Chuck Morris
+  */
   return((drsqr - *rsqr) <= EPSILON ? TRUE : FALSE);
 }
 
-- 
1.9.3

